src/Transformers/FamilyTransformer.php

```diff
@@ -11,13 +11,23 @@ class FamilyTransformer extends TransformerAbstract
 {
     public function transform(Entities\Family $family)
     {
-        return [
-            'id'         => $family->getId()->toString(),
+        $entity = [
+            'id'                  => $family->getId()->toString(),
             'name'                => $family->getName(),
             'paymentStrategy'     => $family->getPaymentStrategy(),
             'completionThreshold' => $family->getCompletionThreshold(),
             'weekStartDay'        => $this->transformWeekStartDay($family->getWeekStartDay()),
         ];
+
+        if ($family->getCreatedDate()) {
+            $entity['createdDate'] = $family->getCreatedDate()->format(\DateTime::ATOM);
+        }
+
+        if ($family->getUpdateDate()) {
```

Comment:
Let's change the contract back to not-nullable and call the `updateCreatedDate` etc methods explicitly in tests.